linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
```

Comment:
1. Can you please open an issue with an example of this happening and steps to replicate? If that's legit I would raise it over on the flow project.
1. As I explained, SublimeLinter doesn't make use of of the column ending. `near` (along with the rest of SublimeLinter :wink:) is very well documented [here](http://www.sublimelinter.com/en/latest/linter_attributes.html#regex)

I don't understand why you've gotten rid of the `line` capture group and the end of string character, and escaping the first `/` isn't necessary.

The file extension check is useful because, currently regardless of the linter's configuration, flow checks all files in a folder that it feels like so it makes sense to ignore anything unexpected.

It looks like you're capturing the full file path which will make the message too long in many cases so the user won't be able to see the part that is actually useful.

Please open an issue with an example of a multi-line error and steps to replicate.